broker/lib/jobs/BnRStatusPollerJob.js

```diff
@@ -126,7 +134,104 @@ class BnRStatusPollerJob extends BaseJob {
         Promise.resolve(operationStatusResponse)
       );
   }
+
+  static checkOperationCompletionStatus20(job_data) {
+    const operationName = job_data.operation;
+    const instanceInfo = job_data.operation_details;
+    const instance_guid = instanceInfo.instance_guid;
+    const backup_guid = instanceInfo.backup_guid;
+    const deployment = instanceInfo.deployment;
+    const plan = catalog.getPlan(instanceInfo.plan_id);
+    return Promise.try(() => {
+        if (operationName === 'backup') {
```

Comment:
This does not seem to be right. Why are we getting the state from `directorManager.getServiceFabrikOperationState` ? Backup is no more a servicefabrikoperation now.

Reply:
If needed, we should move the code out of directorManager.

Reply:
Done